### PR TITLE
jira-1092 fix: container teardown errors, more logging for successful teardown

### PR DESCRIPTION
**Pull request by ghao-ooyala**

### CODE REVIEW

File: src/atlantis/manager/rpc/deploy.go [hunk 1]
```diff
@@ -339,8 +342,12 @@ func (e *TeardownExecutor) Execute(t *Task) error {
 			}
 			last, _ := instance.Delete()
 			if last {
+				t.LogStatus("%s is the last one of its kind  [app: %s SHA: %s Env: %s]",
+					tornContainerID, instance.App, instance.Sha, instance.Env)
+   
 				DeleteAppShaFromEnv(instance.App, instance.Sha, instance.Env)
 			}
+			t.LogStatus("successfully teardown %s", tornContainerID)
```
Comment (shantanoo) [inline src/atlantis/manager/rpc/deploy.go]: s/successfully/Successfully/

File: src/atlantis/manager/rpc/deploy.go [hunk 2]
```diff
@@ -339,8 +342,12 @@ func (e *TeardownExecutor) Execute(t *Task) error {
 			}
 			last, _ := instance.Delete()
 			if last {
+				t.LogStatus("%s is the last one of its kind  [app: %s SHA: %s Env: %s]",
```
Comment (shantanoo) [inline src/atlantis/manager/rpc/deploy.go]: s/"%s is the/"'%s' is the/

Comment (shantanoo) [inline src/atlantis/manager/rpc/deploy.go]: Remove extra space after `kind`.

File: src/atlantis/manager/rpc/deploy.go (hunk 1 above)
Comment (taimai) [inline src/atlantis/manager/rpc/deploy.go]: Also need to clear tornContainers for every loop... However we need to return a list of torn containers, so maybe we need a new variable here?

**Pull request merged**